1906-11-12.xml

```diff
@@ -0,0 +1,1438 @@
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Mihael Ebert</editor>
+                <principal>Will Hanley</principal>
+            </titleStmt>
+            <editionStmt>
+                <edition>
+                    <date when="2018-01-17">January 9, 2017 January 17, 2018</date>
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1906-11-12"><name>Thursday</name>, November 12, 1906</date>
+                    <extent><measure unit="pages" quantity="6">6</measure> pages</extent>
+                </bibl>
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                   
+                    <head type="sub">No. <measure quantity="7608">7,608</measure>] ALEXANDRIA, MONDAY, NOVEMBER 12, 1906. [SIX PAGES P.T.
+                        1. </head>
+                    <div type="advert" xml:id="deg-ad-etc01">
+                        <head>The Eastern Telegraph Company, Limited.</head>
+                        <p>This Company's system of submarine telegraph <lb/>cables is the most direct and
+                            quickest means of <lb/>communication from Egypt to Europe, North and <lb/>South America,
+                            East, South and West Africa, <lb/>India, Australia, New Zealand, China and Japan.</p>
+                        <p>To secure quick transmission, telegrams should <lb/>be marked <hi>Via Eastern.</hi></p>
+                        <p>For latest average time to London, see daily <lb/>bulletin in this paper.</p>
+                        <p>STATIONS IN EGYPT: Alexandria, Cairo, <lb/>Suez, Port-Tewfik, Port-Saïd, Suakin. Head
+                            <lb/>Office. London.</p>
+                    </div> <div type="advert" xml:id="deg-ad-sfp01">
+                        <p>Throw Steel Nibs Away!</p>
+                        <p>They waste time, are costly to use, wear out, scratch, etc.</p>
+                        <p>Try a "Swan" Fountain Pen.</p>
+                        <p>Prices from 10/6</p>
+                        <p>Sold by all Stationers and Importers in Cairo and Alexandria.</p>
+                        <p>Catalogue free.</p>
+                        <p>Mable, Todd &amp; Bard, High Holborn, London.</p>
+                    </div>
+                </div>
+                <div type="section" scope="advertisement"> 
+                    <cols n="3"/>
+                    <cb n="1"/>
+                </div>  <div type="advert" xml:id="deg-ad-orm01">
```

Comment:
This `</div>` tag, which closes the advertising section, should come after all of these ads, just before the end of the page.

Reply:
Where exactly does this tag go? I tried putting it right after the last ad and it turned the box red. I also tried putting it at the very bottom of the page and it also turned the box red.

Reply:
As long as your remove the tag from line 61, it should work find in both places you list. I can take a look in class tomorrow.